benchmarks/ReactiveUI.Benchmarks/Harness.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using BenchmarkDotNet.Running;
+using Xunit;
+
+namespace ReactiveUI.Benchmarks
+{
+    public class Harness
+    {
+        [Fact]
+        public void Auto_Persist_Benchmark()
+        {
+            BenchmarkRunner.Run<AutoPersistBenchmark>();
+        }
+
+        [Fact]
+        public void Create_Reactive_List_Benchmark()
+        {
+            BenchmarkRunner.Run<CreateReactiveListBenchmark>();
+        }
+
+        [Fact]
+        public void Reactive_List_Operation_Benchmark()
```

Comment:
Can you add my bench marks for the inpc property observable in the framework. In the pr I linked the other day.

Reply:
Added.